content/en/continuous_integration/setup_tests/ruby.md

```diff
@@ -61,16 +61,16 @@ require 'datadog/ci'
 
 Datadog.configure do |c|
```

Comment:
`~> 1` is awkward, should be `>= 1` in that case. I'm more for `gem 'ddtrace', '~> 1.0'`, since a v2 would presumably introduce breaking changes, for which we would have an upgrade guide.

Reply:
Note: this is also present in [AppSec getting started](https://github.com/DataDog/documentation/blob/eae1fd30edf42e1feb780aaecab6fcd2a4ac1b0b/content/en/security_platform/application_security/getting_started/ruby.md#get-started).

Reply:
Changed all versioned `ddtrace` references to `~> 1.0`.

Reply:
Changed all versioned `ddtrace` references to `~> 1.0`.